src/backend/distributed/deparser/citus_ruleutils.c

```diff
@@ -251,7 +252,7 @@ pg_get_sequencedef(Oid sequenceRelationId)
  */
 char *
 pg_get_tableschemadef_string(Oid tableRelationId, bool includeSequenceDefaults,
-							 char *accessMethod)
+							 bool includeUDFDefaults, char *accessMethod)
```

Comment:
Are includeSequenceDefaults and includeUDFDefaults always the same value? We should probably just have 1 flag then.

Reply:
For now, include `UDFdefauluts` is always false. It will change in a separate PR, when I introduce the logic  for having distributed object propagation for metadata synced tables.

I will share a design doc later in the day.